internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -127,18 +132,28 @@ func (e *EnvStackConfig) importedVPC() *template.ImportVPC {
 }
 
 func (e *EnvStackConfig) managedVPC() template.ManagedVPC {
-	if e.in.AdjustVPCConfig != nil {
-		return template.ManagedVPC{
-			CIDR:               e.in.AdjustVPCConfig.CIDR,
-			AZs:                e.in.AdjustVPCConfig.AZs,
-			PublicSubnetCIDRs:  e.in.AdjustVPCConfig.PublicSubnetCIDRs,
-			PrivateSubnetCIDRs: e.in.AdjustVPCConfig.PrivateSubnetCIDRs,
+	defaultManagedVPC := template.ManagedVPC{
+		CIDR:               DefaultVPCCIDR,
+		PublicSubnetCIDRs:  DefaultPublicSubnetCIDRs,
+		PrivateSubnetCIDRs: DefaultPrivateSubnetCIDRs,
+	}
+	// If a manifest is present, it is the only place we look at.
+	if e.in.Mft != nil {
+		if v := e.in.Mft.Network.VPC.ManagedVPC(); v != nil {
+			return *v
 		}
+		return defaultManagedVPC
```

Comment:
> out of curiosity, was there a reason why you preferred the e.in.Network.VPC.ManagedVPC() not to return the default VPC?

Not particularly "preferred". I thought of `Mft` here as "where user input comes in", and thought of `default` as a non-user-input - that was the mindset behind it!

I think one day we will want to let `manifest` package handles the default, eventually removing this middle layer. But I was hesitant to do it here because other contributors might find it confusing that sometimes the defaults is handled in `manifest` while other times in in `stack`.

> What do you think of this method signature?
```
if vpc := Mft.Network.VPC.ManagedVPC(); vpc != nil {
   return *vpc
}
```
I felt like if we are letting `manifest` handle the defaults here, then it's probably okay if we just return `vpc` without the `bool` because `stack` wouldn't need that info. What do ya think?